{"name": "test_wrong_pwd", "status": "passed", "description": "密码输入6次错误", "attachments": [{"name": "log", "source": "922292c5-0c95-48f8-9386-c3023219316c-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "42ed28d3-cff3-4598-891f-7dabbf5b5c04-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "568606fb-a50b-4189-939e-ad02a5633557-attachment.txt", "type": "text/plain"}], "start": 1632453017017, "stop": 1632453019150, "uuid": "6bd69418-aaf0-4aa4-b2de-9348939d3a45", "historyId": "7b9480ebb1961c7c14cb69cc7e65ba50", "testCaseId": "e4c5762512a76c681290f066bcc0e861", "fullName": "test_case.Login_case.TestLogin#test_wrong_pwd", "labels": [{"name": "severity", "value": "critical"}, {"name": "story", "value": "参数填写不正确，登录失败"}, {"name": "feature", "value": "登录模块"}, {"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "Login_case"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-KSOAST3"}, {"name": "thread", "value": "14540-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.Login_case"}]}